src/handlers/localstorage/ens.ts

```diff
@@ -33,6 +35,12 @@ export const saveNameFromLabelhash = (
   value: string | Record<string, unknown>
 ) => saveGlobal(ensLabelhashesKey(key), value, ensProfileVersion);
 
+export const getENSName = async (key: EthereumAddress) =>
+  await getGlobal(ensNameKey(key), '', ensProfileVersion);
+
+export const saveENSName = (key: EthereumAddress, value: string) =>
+  saveGlobal(ensNameKey(key), value, ensProfileVersion);
```

Comment:
Users can have multiple wallets and an ENS domain associated with each, right? Does this account for that?